src/lib.rs

```diff
@@ -29,14 +28,11 @@ mod magic;
 mod parent;
 
 /// Convenience identifier for an unknown MIME type.
-pub static UNKNOWN_TYPE: &str = "application/octet-stream";
+pub const UNKNOWN_TYPE: Mime = mime::APPLICATION_OCTET_STREAM;
 
 /// Convenience identifier for the MIME type for an empty file.
 pub static EMPTY_TYPE: &str = "application/x-zerosize";
```

Comment:
It's a bit weird to have one of the identifiers be a `Mime` and another one be a `&str`.

Reply:
Oh, yeah, I'm not very happy about those.

The mime crate uses an internal Atom mechanism to define constants, but it is not exposed to the outside, so we can't define `EMPTY_TYPE` as a constant of type `Mime`.  I guess we could have an `empty_type()` function.

As for `UNKNOWN_TYPE`... is it worth having?  The only place it is used is in the fallback for `get_mime_types_from_file_name`.  It is nice that it returns an actual MIME-type that indicates "I don't know which type it is", but would it be better/equivalent to just return an empty array of MIME-types?

Reply:
Yeah, I don't think these constants add much, at this point. They are there in C for convenience, but they never make it out of xdgmime anyway, so I'm happy with a commit that removes them altogether.

Reply:
Oh?  I'll remove the function I exported for empty_file...